File: Frontend/src/features/player/services/playerService.ts
import { http } from "../../../http";

class PlayerService{
    async getUrlGame(gameId: string){
        try{
            const response = await http.get(`player/get-${gameId}`);
            return response.data;
        }catch(error){
            return error;
        }
    }
}

export const playerService = new PlayerService();

File: Backend/Services/Player/PlayerService.cs
using System.Security.Claims;
using Backend.Repositories.Player;

namespace Backend.Services.Player;

public class PlayerService
{
    private PlayerRepository _playerRepository;
    
    public PlayerService(PlayerRepository playerRepository)
    {
        _playerRepository = playerRepository;
    }

    public async Task<string?> GetGameUrl(Guid gameId)
    {
        return await _playerRepository.GetUrlByGameId(gameId);
    }
}